frameworks/hdfs/tests/test_tls.py

```diff
@@ -1,16 +1,23 @@
 import pytest
+import retrying
+
 
 import sdk_cmd
-import sdk_install
 import sdk_hosts
+import sdk_install
 import sdk_plan
+import sdk_tasks
 import sdk_utils
-import retrying
+
 
 from security import transport_encryption
 
 from tests import config
 
+pytestmark = pytest.mark.skipif(sdk_utils.dcos_version_less_than("1.10"),
```

Comment:
Why assign to a value if it's unused?

Reply:
This is the way pytest handles global marks. They have to be called `pytestmark`. There is an error here though, they should be a list and not individually assigned.

Reply:
(I also messed up the enterprise conditional 😞)